charts/ibm-mqadvanced-server-dev/templates/stateful-set.yaml

```diff
@@ -24,8 +24,10 @@ metadata:
 spec:
   serviceName: {{ .Values.service.name }}
   replicas: 1
+  {{- if and (ge (.Capabilities.KubeVersion.Major | int) 1) (ge (.Capabilities.KubeVersion.Minor | int) 7) }}
```

Comment:
This looks like a check to make sure the Kube version has a minor of 1.7 but the above line in the readme says 1.6 now.